web/src/modules/layout/components/Footer.js

```diff
@@ -38,10 +37,7 @@ const FooterContainer = styled.footer`
  */
 class Footer extends React.PureComponent<PropsType> {
   static getVersion (): Node {
-    if (buildConfig().development) {
-      return <span>{__VERSION__}</span>
-    }
-    return null
+    return <span>{__VERSION__}</span>
```

Comment:
I think we can show it always. I kind of this it is nice to know this in production aswell

Reply:
Did you check how it looks on mobile? To be honest I'm not sure if that is really a good idea, I don't remember seeing this on other sites.

Reply:
This is just a unnecessary information for all our users and only useful for us, so I don't think we should show it.

Reply:
Apart from that the version is not correct if the version got bumped before. The version bump is just made in git and not to the files in the CI, but we just read the version from the files in the webpack config. So if there was a version bump and we create a new production release, it shows an old version (the previous one).